pkg/build/graph/analysis/bc.go

```diff
@@ -238,6 +238,19 @@ func FindPendingTags(g osgraph.Graph, f osgraph.Namer) []osgraph.Marker {
 	return markers
 }
 
+// getImageStreamTagSuggestion will return the appropriate marker Suggestion for when a BuildConfig is missing its input ImageStreamTag;  in particular,
+// it will determine whether or not another BuildConfig can produce the aforementioned ImageStreamTag
+func getImageStreamTagSuggestion(g osgraph.Graph, f osgraph.Namer, tagNode *imagegraph.ImageStreamTagNode, bcNode *buildgraph.BuildConfigNode) osgraph.Suggestion {
+	bcs := []string{}
+	if inputBCNode, points := buildPointsToTag(g, tagNode); points {
+		bcs = append(bcs, f.ResourceName(inputBCNode))
```

Comment:
isn't this still just going to return the first bc that points to tagNode, rather than all of them?